packer/feeds.json

```diff
@@ -0,0 +1,44 @@
+{
+    "builders": [{
+        "type": "amazon-ebs",
+        "region": "us-east-2",
+        "source_ami_filter": {
+            "filters": {
+                "virtualization-type": "hvm",
+                "name": "debian-stretch-hvm-x86_64-gp2-*",
+                "root-device-type": "ebs"
+            },
+            "owners": [
+                "379101102735"
+            ],
+            "most_recent": true
+        },
+        "instance_type": "t2.micro",
+        "ssh_username": "admin",
+        "ami_name": "cyhy-feeds-hvm-{{timestamp}}-x86_64-ebs",
+        "ami_regions": [
+            "us-east-1",
+            "us-west-1",
+            "us-west-2"
+        ],
+
+        "tags": {
+            "Team": "NCATS OIS - Development",
+            "Application": "Cyber Hygiene",
+            "OS_Version": "Debian Stretch",
+            "Release": "Latest",
+            "Base_AMI_Name": "{{ .SourceAMIName }}"
+        }
+    }],
+
+    "provisioners": [
+        {
+            "type": "ansible",
+            "playbook_file": "packer/ansible/playbook.yml",
+            "groups": [
+                "debian",
```

Comment:
`debian` shouldn't be here.  The packer code was updated not to require it.  I think you were working from an old version.  Please update.